.gitignore

```diff
@@ -331,3 +331,4 @@ $RECYCLE.BIN/
 .now
 
 # End of Custom
+.yarn
```

Comment:
?

Reply:
Shouldn't. Are you using [`yarn classic`](https://classic.yarnpkg.com/lang/en/)?

Reply:
yarn 3